### PR TITLE
fix: create an order event redirects to first page on refresh (FPLA-2561)

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2561

### Change description ###
PageShowCondition updates within the 'create an order' event to ensure refresh redirects page to order type selection page (page 1)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "PageID": "SingleCareOrder",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND singleCareOrder_label!=\"\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" AND orderTypeAndDocument.type!=\"\" AND singleCareOrder_label!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:76]: is it needed? orderTypeAndDocument.type=\"DISCHARGE_OF_CARE_ORDER\" should be enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 2]
```diff
@@ -245,7 +246,7 @@
     "PageID": "OrderTitleAndDetails",
     "PageDisplayOrder": 7,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"BLANK_ORDER\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"BLANK_ORDER\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:249]: orderTypeAndDocument.type=\"BLANK_ORDER\" is enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 3]
```diff
@@ -271,7 +272,7 @@
     "PageID": "EPOChildren",
     "PageDisplayOrder": 8,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:275]: orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" is enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 4]
```diff
@@ -284,7 +285,7 @@
     "PageID": "EPOType",
     "PageDisplayOrder": 9,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:288]: orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" is enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 5]
```diff
@@ -311,7 +312,7 @@
     "PageID": "EPOPhrase",
     "PageDisplayOrder": 10,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:315]: orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" is enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 6]
```diff
@@ -324,7 +325,7 @@
     "PageID": "EPOEndTime",
     "PageDisplayOrder": 11,
     "PageColumnNumber": 1,
-    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:328]: orderTypeAndDocument.type=\"EMERGENCY_PROTECTION_ORDER\" is enough

File: ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json [hunk 7]
```diff
@@ -380,7 +381,7 @@
     "PageDisplayOrder": 13,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y",
-    "PageShowCondition": "orderTypeAndDocument.type=\"UPLOAD\"",
+    "PageShowCondition": "orderTypeAndDocument.type=\"UPLOAD\" AND orderTypeAndDocument.type!=\"\"",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToFields/CareSupervision/createOrder/createOrder.json:384]: orderTypeAndDocument.type=\"UPLOAD\" is enough

**Review by tomaszpowroznik — APPROVED** [review]

**Review by joez-f — APPROVED** [review]

**Pull request merged**